### PR TITLE
[105]  MAIN - Updated README to explain consequences of enabling checksums

### PR DESCRIPTION
**Pull request by JustinKyleJames**
Updated README to explain consequences of enabling checksums on uploads.

### CODE REVIEW

File: README.md
Comment (trel) [inline README.md:344]:
```suggestion
Due to this, uploads with checksums can be delayed while the checksum is being calculated in the client. In addition, performing the open/read/close will cause additional iRODS policy to fire.
```

File: README.md
Comment (alanking) [inline README.md:341]: I feel like these need to be combined, or reason 1 needs to be dropped. Reason 2 feels like the explanation for reason 1 to me. Thoughts?

Comment (trel) [reply README.md:341]: kind of...

1 is the lack of an appropriate singular API call.
1 is the lack of iRODS support for all of the Globus-supported algorithms.

Comment (alanking) [reply README.md:341]: So `rcDataObjChksum` doesn't do what we want? Do we need to make a new API? I guess I'm left wanting a "why" for reason 1.

Comment (trel) [reply README.md:341]: I think it's because you can't select your checksum algorithm with `rcDataObjChksum`.

File: README.md
Comment (JustinKyleJames) [reply README.md]: I made that change.

File: README.md
Comment (trel) [reply README.md]: looks better i think

Comment (alanking) [reply README.md]: Yes, looks better.

**Comment by JustinKyleJames** [comment]
> I think this is ready to squash if you're happy with it. Please also link #106 in the commit title since this also explains the issue raised there.

Done

**Review by korydraughn — APPROVED** [review]
Based on the review history, this appears to be ready.

Let's wait for one more approval before pounding.

**Review by trel — APPROVED** [review]
looks good, # it.

**Comment by JustinKyleJames** [comment]
> looks good, # it.

done

**Comment by korydraughn** [comment]
Merging.

**Pull request merged**
